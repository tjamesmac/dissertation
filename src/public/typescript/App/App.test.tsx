import { shallow } from 'enzyme';
import * as React from 'react';
import renderer from 'react-test-renderer';
import { App } from './App';

describe('App component renders correctly', () => {
  it('renders as expected', () => {
    const result = shallow(<App/>);
    expect(result.exists()).toBe(true);
  });
  it('matches snapshot', () => {
    const tree = renderer
      .create(<App />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
