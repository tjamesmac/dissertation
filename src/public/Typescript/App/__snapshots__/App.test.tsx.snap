// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders correctly matches snapshot 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-12"
    >
      <form
        onSubmit={[Function]}
      >
        <div
          className="row"
        >
          <div
            className="col-12"
          >
            <label>
              Please enter a gender
            </label>
            <input />
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col-12"
          >
            <label>
              Please enter an advert
            </label>
            <div
              contentEditable={true}
              id="textarea"
              onChange={[Function]}
            />
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col-4"
          >
            <button>
              Click me
            </button>
          </div>
        </div>
      </form>
    </div>
  </div>
</div>
`;
